### PR TITLE
enable AWS (us-east-2)

### PR DESCRIPTION
**Pull request by goneri**
enable AWS (us-east-2)
    
Enable the AWS driver and set the credential for the zuul user.

### CODE REVIEW

File: nodepool/openstack/clouds.yaml.j2 [hunk 1]
```diff
@@ -7,6 +7,7 @@ cache:
     port: 5
 
 clouds:
+  __nodepool_clouds_yaml_aws_access_key_id
```
Comment (pabelanger) [inline nodepool/openstack/clouds.yaml.j2:10]: why do we need aws credentials in clouds.yaml?

Comment (goneri) [reply nodepool/openstack/clouds.yaml.j2:10]: ahah, lets me fix that.

File: ansible/group_vars/nodepool.yaml [hunk 2]
```diff
@@ -158,3 +158,30 @@ __nodepool_clouds_yaml_rdocloud_username: !vault |
   32616265313362306338396566333835616665613063653830613566346634336265396666626138
   3030353531633531390a316166663939393236393765653532613236643963656262383632613437
   6335
+
+# windmill.boto3
+goto3_user_name: nodepool
```
Comment (pabelanger) [inline ansible/group_vars/nodepool.yaml:163]: boto3?

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**